README.md

```diff
@@ -10,24 +10,45 @@
 - Licensed under: **MIT License**
 
 ## Dependencies
-- Python: >= 2.7 or Python: >= 3.3
+- Python (tested against): >= 2.7 or Python: >= 3.5
 - [requests](http://docs.python-requests.org/en/latest/)
 
 ## Installation
-### from Python Package Index (Pypi)
+We recommend using a virtualenv, but nothing prevents you installing it into
+the root system.
+
+### From Python Package Index (Pypi)
 [Pybooru on Pypi.](https://pypi.python.org/pypi/Pybooru/)
 
 ```bash
-pip install --user Pybooru
+pip install Pybooru
 ```
 
-### Manual installation
+### Installing for development
```

Comment:
Maybe it would be a good idea to provide installation instructions for pipenv?

Reply:
Ahh, I thought you use pipenv only for the travis because the previous instructions didn't use pipenv either. The thing though, that it is not easy to work with multiple versions of python with pipenv. There is a problem though, pylint has no version which is supports py2.7 up to py3.8, meaning as long as `Pipfile.lock` (or `requirements.txt` for that matter) is used to define `pylints` version the installation will fail. 
I can, however, advise the user to use `--skip-lock` in either one of the versions or even always. 

Though I don't really see the advantage of using `pipenv` in general for development. In a test environment it is nice because it automatically creates the virtualenv. But I guess most developers use either `pyenv` or just create their venv themselves. And in case one wants a version pinning one could use something like `requirements-py3.txt` and `requirements-py2.txt`. But then again even in the tests I am not so sure about version pinnings because you wouldn't see anything when a required package has changed.